README.md

```diff
@@ -26,30 +26,23 @@ The [`cozy-desktop`](https://github.com/cozy-labs/cozy-desktop) repository is wh
 If you are interested in fixing issues and contributing directly to the code base,
 please see the document [How to Contribute](https://github.com/cozy-labs/cozy-desktop/wiki/How-to-Contribute), which covers the following:
 
--   [Setting up a development environment](doc/developer/setup.md)
+-   [How to build and run from source](doc/developer/setup.md)
```

Comment:
Right now `doc/developer/setup.md` targets developers who want to hack on the code, but not really users who want to build from source e.g. because there is no available package for their plateform.

Maybe we should wait until it is reorganized before renaming the link? Wdyt?

Reply:
I picked the formulation from vscode https://github.com/Microsoft/vscode/blob/master/README.md#contributing

It will change when we will work on the dev doc.

Reply:
:ok_hand: